client/components/external-link/index.jsx

```diff
@@ -73,9 +73,17 @@ class ExternalLink extends Component {
 				{ this.props.icon && this.props.showIconFirst && iconComponent }
 				{ this.props.children }
 				{ this.props.icon && ! this.props.showIconFirst && iconComponent }
+				{ this.props.icon && (
+					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+					<span className="screen-reader-text">
+						{
+							/* translators: accessibility text */
+							'(opens in a new tab)'
```

Comment:
This needs to be wrapped in `translate( '(opens in a new tab)' )` or otherwise it won't be translated.

Reply:
my bad. I missed that while reviewing. thx for spotting.

@lcollette can you address this issue when you can? thx

Reply:
Got it!

Reply:
No worries, happens. It'll be a simple fix. :-)